laa_court_data_api_app/routers/hearing_summaries.py

```diff
@@ -11,38 +11,38 @@
 from laa_court_data_api_app.models.prosecution_cases.prosecution_cases import ProsecutionCases
 from laa_court_data_api_app.models.prosecution_cases.prosecution_cases_results import ProsecutionCasesResults
 
-logger = logging.getLogger(__name__)
+logger = structlog.get_logger(__name__)
 router = APIRouter()
 
 
 @router.get("/v2/hearingsummaries/{urn}", response_model=HearingSummariesResponse, status_code=200)
 async def get_hearing_summaries(urn: str):
     logger.info("Calling_Hearing_Summaries_Get_Endpoint")
-    logging.info(f"Hearing_Summaries_Get_{urn}")
+    logger.info("Hearing_Summaries_Get", urn=urn)
     client = CourtDataAdaptorClient()
     cda_response = await client.get("/api/internal/v2/prosecution_cases",
                                     params={"filter[prosecution_case_reference]": urn})
 
     if cda_response is None:
-        logging.error("Prosecution_Case_Endpoint_Did_Not_Return")
+        logger.error("Prosecution_Case_Endpoint_Did_Not_Return")
         return Response(status_code=424)
 
     match cda_response.status_code:
         case 200:
-            logging.info("Prosecution_Case_Endpoint_Returned_Success")
+            logger.info("Prosecution_Case_Endpoint_Returned_Success")
             prosecution_case_results = ProsecutionCasesResults(**cda_response.json())
             summaries = map_hearing_summaries(prosecution_case_results.results)
-            logging.info(f"Hearing_Summaries_To_Show: {summaries.count}")
+            logger.info("Hearing_Summaries_To_Show", count=len(summaries))
             return HearingSummariesResponse(hearing_summaries=summaries,
                                             overall_defendants=map_defendant_list(prosecution_case_results.results))
         case 400:
-            logging.info("Prosecution_Case_Endpoint_Validation_Failed")
+            logger.info("Prosecution_Case_Endpoint_Validation_Failed")
```

Comment:
Could add status code to help with tracing through logs on incident

Could also be a `warn` for a bad request